org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java

```diff
@@ -1021,7 +1035,48 @@ else if (DisplayType.isDate(iPara.getDisplayType()))
 			}
 		}				
 	}
-	
+
+	private static final String displayColumns = "Name,Value,DocumentNo";
+	private String getDisplay(MPInstance i, MPInstancePara ip, Object id) {
+		try {
+			MProcessPara pp = MProcess.get(i.getAD_Process_ID()).getParameter(ip.getParameterName());
+			MTable table = null;
+			String column = "";
+
+			if (pp != null) {
+				if (pp.getAD_Reference_ID() == DisplayType.Search || pp.getAD_Reference_ID() == DisplayType.TableDir)
```

Comment:
this checking is not correct, DisplayType.Search can have AD_Reference_Value_ID too.